### PR TITLE
fix default value of epsilon in l2_normalize func

### PR DESCRIPTION
**Pull request by shippingwang**
fix 2nd of [PaddlePaddle/FluidDoc#672](https://github.com/PaddlePaddle/FluidDoc/issues/672)

### CODE REVIEW

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -4731,7 +4731,7 @@ def l2_normalize(x, axis, epsilon=1e-12, name=None):
             the dimension to normalization is rank(X) + axis. -1 is the
             last dimension.
         epsilon(float): The epsilon value is used to avoid division by zero, \
-            the defalut value is 1e-10.
+            the defalut value is 1e-12.
```
Comment (shanyi15) [inline python/paddle/fluid/layers/nn.py]: default

Comment (shippingwang) [reply python/paddle/fluid/layers/nn.py]: ：）copy that

**Review by shanyi15 — DISMISSED** [review]
LG comments

**Comment by shippingwang** [comment]
already done by #16805

**Pull request closed without merging**